tests/OSPSuite.Starter/OSPSuite.Starter.csproj

```diff
@@ -36,25 +31,24 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-
   <ItemGroup>
     <None Include="..\..\LICENSE">
       <Pack>True</Pack>
-      <PackagePath></PackagePath>
+      <PackagePath>
+      </PackagePath>
     </None>
     <None Include="..\..\src\OSPSuite.Dimensions\OSPSuite.Dimensions.xml" Link="OSPSuite.Dimensions.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-
   <ItemGroup>
-    <PackageReference Include="OSPSuite.DevExpress" Version="21.2.15" />
-    <PackageReference Include="OSPSuite.FuncParser" Version="4.0.0.73" GeneratePathProperty="true" />
-    <PackageReference Include="OSPSuite.SimModel" Version="4.0.0.75" GeneratePathProperty="true" />
+    <PackageReference Include="DevExpress.Win.Design" Version="21.2.15" />
```

Comment:
That's needed by VS designer only. If you don't have it, then you can't design. I wonder if it makes sense to **not** deploy that in OSPSuite.DevExpress and to include it only for local dev work, and how that could work. The package includes a lot of assemblies that are probably not needed.

Basically include the package ref on your local machine, but not in CI builds.